resources/scripts.js

```diff
@@ -0,0 +1,13 @@
+// Function to make header stick as user scrolls
+function sticky-header() {
+    if (window.pageYOffset > sticky) {
+        header.classList.add("sticky");
+    } else {
+        header.classList.remove("sticky");
+    }
+}
+
+// Variables and execution for "sticky-header" function
+var header = document.getElementById("nav-bar");
```

Comment:
`var` should not be used.  If the variable has no chance of being redeclared then `const` is preferable.